ansible/roles/openfisca-api/tasks/main.yml

```diff
@@ -0,0 +1,103 @@
+- name: Install OS packages
+  ansible.builtin.apt:
+    install_recommends: no
+    name:
+      - acl # Provides "setfacl" command, used by Ansible to become another Unix user
+      - nginx
+      - python3.7 # cf https://github.com/openfisca/openfisca-core#environment
+      - python3-pip
+      - python3-virtualenv
+    state: present
+    update_cache: yes
+
+- name: Create Unix group for OpenFisca
+  ansible.builtin.group:
+    name: "{{ unix_group_name }}"
+    state: present
+
+- name: Create Unix user for OpenFisca Web API
+  ansible.builtin.user:
+    name: "{{ unix_user_name }}"
+    group: "{{ unix_group_name }}"
+  register: unix_user
+
+- name: Define virtualenv directory from Unix user home directory
+  ansible.builtin.set_fact:
+    venv_dir: "{{ unix_user.home }}/venv"
+
+- name: Install latest version of OpenFisca Web API in virtualenv with OpenFisca-France
+  ansible.builtin.pip:
+    # Using chdir to solve an error as explained by [this comment](https://github.com/ansible/ansible/issues/22967#issuecomment-500604054)
+    chdir: "{{ venv_dir }}"
+    name:
+      - OpenFisca-Core[web-api]
+      - OpenFisca-France
+    state: latest
+    virtualenv: "{{ venv_dir }}"
+    virtualenv_site_packages: no
+  become_user: "{{ unix_user_name }}"
+
+- name: Copy systemd service file
+  ansible.builtin.template:
+    src: systemd/openfisca-web-api-fr.service.j2
+    dest: /etc/systemd/system/openfisca-web-api-fr.service
+
+- name: Enable and start systemd service
+  ansible.builtin.systemd:
+    daemon_reload: yes
+    enabled: yes
+    state: restarted
+    name: openfisca-web-api-fr
+
+- name: Check that OpenFisca Web API is actually started
+  ansible.builtin.uri:
+    return_content: yes
+    status_code: 300
+    url: "http://localhost:{{ http_port }}"
+  register: this
+  failed_when: this.json.welcome != welcome_message
```

Comment:
> After creating commit 767fb8234cf5587fc6944208375e5d28ae448072, I noticed that the welcome message is actually specified by a variable of the Ansible role, so it's not so fragile since there is a single source of truth.
>
> However, I'm OK to drop that check for the reason it's not specified, and I would prefer not relying on a feature provided by courtesy.